pkg/controller.v1/tensorflow/pod.go

```diff
@@ -240,6 +241,19 @@ func (tc *TFController) createNewPod(tfjob *tfv1.TFJob, rt, index string, spec *
 	return nil
 }
 
+func checkPodTTL(tfjob *tfv1.TFJob) bool {
+	if tfjob.Status.CompletionTime == nil {
+		return true
+	}
+	if value, ok := tfjob.Annotations[TFJobWaitingWorkerAnnotation]; ok && value == "true" {
```

Comment:
I think the value is not bool, it should be integer.

Reply:
Done